tools/nxstyle.c

```diff
@@ -477,7 +543,7 @@ int main(int argc, char **argv, char **envp)
           (strcmp(line, " * Private Functions\n") == 0 ||
            strcmp(line, " * Public Functions\n") == 0))
         {
-          //fprintf(stderr, "Functions begin at line %d:%d\n", lineno, n);
+          //ERROR("Functions begin", lineno, n);
           bfunctions = true;
```

Comment:
Same as above:  Remove or use #if 0 with comment.  No C++ // comments.

Reply:
ehmm @patacongo  that came from master